Xamarin.Forms.Platform.Android/CellAdapter.cs

```diff
@@ -81,15 +81,15 @@ internal AView ContextView
 		public bool OnActionItemClicked(ActionMode mode, IMenuItem item)
 		{
 			OnActionItemClickedImpl(item);
-			if (mode != null && mode.Handle != IntPtr.Zero)
-				mode.Finish();
+			_actionMode?.Finish();
```

Comment:
that was a hack because we were checking the wrong thing.. if you look at all the code you will see _actionMode is set to null on destroy and thats the mode that we created when clicking the toolbar item.